Documentation/help_files/openemr_installation_help.php

```diff
@@ -0,0 +1,589 @@
+                    
+                    <p><?php echo xlt("In addition during the installation process certain application files are modified");?>.
+                    
+                    <p><?php echo xlt("These files need to have their permissions set so that they could be modified during the installation process");?>.
+                    
+                    <p><?php echo xlt("These two files should be writable by everybody"); ?>.
+                        <ul>
+                            <li><strong><?php echo xlt("openemr/sites/default/sqlconf.php"); ?></strong></li>
+                            <li><strong><?php echo xlt("openemr/interface/modules/zend_modules/config/application.config.php"); ?></strong></li>
+                        </ul>
+                    <p><?php echo xlt("In Linux use the following command to set the proper permissions");?>.
+                    
+                    <p><code>sudo chmod 666 filename</code>
+                    
+                    <p><?php echo xlt("This will allow all users to read and write to the file but not execute it.");?>
+                    
+                    <p><?php echo xlt("In the day to day working of the application certain files are created by the application and stored in the web server document root directory. Files like faxes, images etc. are constantly being uploaded to the application");?>.
+                    
+                    <p><?php echo xlt("To facilitate this process the following directories should be writable by the web server (Apache)"); ?>:
+                        <ul>
+                            <li><strong>/var/www/html/openemr/sites/default/documents</strong> - <?php echo xlt("where all scanned documents etc are stored"); ?></li>
+                            <li><strong>/var/www/html/openemr/sites/default/edi</strong> - <?php echo xlt("saves transmitted electronic batches for various kinds"); ?></li>
+                            <li><strong>/var/www/html/openemr/sites/default/era</strong> - <?php echo xlt("saves output from automated remittance (X12 835) processing. This includes archived remittance files, and copies of the generated HTML reports"); ?></li>
+                            <li><strong>/var/www/html/openemr/sites/default/letter_templates</strong> - <?php echo xlt("sample empty letter template"); ?></li>
+                            <li><strong>/var/www/html/openemr/gacl/admin/templates_c</strong> - <?php echo xlt("Smarty templates directory for gacl (Generic Access Control Lists)"); ?></li>
+                            <li><strong>/var/www/html/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled</strong> - <?php echo xlt("Smarty templates directory for the calendar"); ?></li>
+                            <li><strong>/var/www/html/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/cache</strong> - <?php echo xlt("Smarty templates directory for the calendar"); ?></li>
```

Comment:
Recommend customizing the paths above (if possible), like is done in the setup.php script.